src/django_components/app_settings.py

```diff
@@ -115,5 +120,10 @@ def _validate_context_behavior(self, raw_value: ContextBehavior) -> ContextBehav
             valid_values = [behavior.value for behavior in ContextBehavior]
             raise ValueError(f"Invalid context behavior: {raw_value}. Valid options are {valid_values}")
 
+    # TODO - ADD TEST
+    @property
+    def TAG_FORMATTER(self) -> Union[str, Type["TagFormatterABC"]]:
```

Comment:
When it comes to using the `TagFormatter` by users, the flow could look like this:
1. User defines their own `TagFormatter`, which inherits from the `TagFormatterABC`, and defines the `format_*` and `parse_*` methods, e.g. like the [ShorthandTagFormatter](https://github.com/EmilStenstrom/django-components/pull/572/files#diff-d49ecbc3623a4ed5408729717a2ce020b9f336330a24a90cc7e75da2ea7416e1R211) does.
2. They set their tag formatter via settings, either as import path:
	```py
	COMPONENTS = {
		"tag_formatter": "path.to.MyTagFormatter",
	}
	```
	Or by reference
	```py
	from path.to import MyTagFormatter

	COMPONENTS = {
		"tag_formatter": MyTagFormatter,
	}
	```
3. Profit